common4j/build.gradle

```diff
@@ -203,6 +209,7 @@ idea {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib:${rootProject.ext.kotlinVersion}"
+    implementation "androidx.annotation:annotation:1.4.0"
```

Comment:
You absolutely cannot do this - cannot add an Android dependency to a non-Android project. `common4j` is a non-Android module

Reply:
This dependency has been removed.